### PR TITLE
New package: angeldev0.messyorganizer version 1.0.0

### PR DESCRIPTION
**Pull request by 4ngel2769**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156605)

### CODE REVIEW

**Comment by 4ngel2769** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-06-05T10:42:10.1606738Z ##[error]Manifest Validation Failed
 2024-06-05T10:42:15.2279985Z ##[section]Finishing: Validate Manifest

(Automated response - build 883.)

File: manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- organuzer
+- organizer
+- file organizer
+- messyorganizer
+InstallationNotes: 'Thanks for installing MessyOrganizer! If you find any bugs in our app, please report them here: https://github.com/4ngel2769/messy-file-organizer/issues/new'
+Icons:
+- IconUrl: https://raw.githubusercontent.com/4ngel2769/messy-file-organizer/main/mfo.png
+  IconFileType: png
+  IconResolution: 256x256
+  IconTheme: default
```
Comment (stephengillie) [inline manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.locale.en-US.yaml:30]:
```suggestion
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @4ngel2769,

Our Veriifed Publisher system is new. So new that I'm not sure of most of the details. For now, is it okay if the manifest proceeds without the icons?

File: manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.locale.en-US.yaml (hunk 1 above)
Comment (Trenly) [reply manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.locale.en-US.yaml:30]: As a note - I don't believe the message from the pipelines is accurate; It isn't that it requires VerifiedPublishers, but rather that it is not allowed by policy in this repo. Its just that the framework for VerifiedPublishers (which hasn't been finished yet) has been used to implement the policy

File: manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: angeldev0.messyorganizer
+PackageVersion: 1.0.0
+InstallerType: inno
+Installers:
+- InstallerUrl: https://github.com/4ngel2769/messy-file-organizer/releases/latest/download/MessyFileOrganizer-setup-x64.exe
```
Comment (Trenly) [inline manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.installer.yaml:8]:
```suggestion
- InstallerUrl: https://github.com/4ngel2769/messy-file-organizer/releases/download/v1.0.0-beta.1/MessyFileOrganizer-setup-x64.exe
```
Using a versioned URL is preferred to using a `latest` URL, to ensure the package doesn't encounter hash mismatch errors when a new version is released

File: manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: angeldev0.messyorganizer
+PackageVersion: 1.0.0
+InstallerType: inno
+Installers:
+- InstallerUrl: https://github.com/4ngel2769/messy-file-organizer/releases/latest/download/MessyFileOrganizer-setup-x64.exe
+  Architecture: neutral
```
Comment (Trenly) [inline manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.installer.yaml:9]:
```suggestion
  Architecture: x64
```

File: manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: angeldev0.messyorganizer
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Angel C
```
Comment (Trenly) [inline manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.locale.en-US.yaml:7]:
```suggestion
Publisher: Angel Dev0
```
This should match what is written to Control Panel to ensure WinGet can detect the application once it is installed

File: manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: angeldev0.messyorganizer
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Angel C
+PublisherUrl: https://github.com/4ngel2769
+PublisherSupportUrl: https://github.com/4ngel2769
```
Comment (Trenly) [inline manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.locale.en-US.yaml:9]:
```suggestion
PublisherSupportUrl: https://github.com/4ngel2769/messy-file-organizer/issues
```
This is generally preferred to be the link where users can submit new support requests or file new issues

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by 4ngel2769** [comment]
> Hi @4ngel2769,
> 
> Our Veriifed Publisher system is new. So new that I'm not sure of most of the details. For now, is it okay if the manifest proceeds without the icons? 

Yeah, the icons are optional anyway.

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 883.)

**Comment by stephengillie** [comment]
Hi @4ngel2769,

It seems that @vikingnope already submitted this package for you: 

- https://github.com/microsoft/winget-pkgs/pull/156596

**Comment by 4ngel2769** [comment]
> Hi @4ngel2769,
> 
> It seems that @vikingnope already submitted a PR for this package for you:
> 
> * [New package: 4ngel2769.MessyOrganizer version 1.0.0-beta.1 #156596](https://github.com/microsoft/winget-pkgs/pull/156596)

It appears to be the case. What now?

**Comment by vikingnope** [comment]
You can just close this PR

> > Hi @4ngel2769,
> > It seems that @vikingnope already submitted a PR for this package for you:
> > 
> > * [New package: 4ngel2769.MessyOrganizer version 1.0.0-beta.1 #156596](https://github.com/microsoft/winget-pkgs/pull/156596)
> 
> It appears to be the case. What now?

**Pull request closed without merging**